docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/job-manager.md

```diff
@@ -4,9 +4,9 @@ The job here refers to the long tasks running at the storage layer. For example,
 
 ## Statements List
 
-### submit job compact / flush
+### SUBMIT JOB COMPACT
 
-The `submit job compact/flush` statement creates a new job and returns the job ID in the job manager, and executes the `compact/flush` command in the storage. The example is as follows:
+The `SUBMIT JOB COMPACT` command triggers the long time-consuming `RocksDB compact` operation. The example returns the results as follows:
```

Comment:
The `SUBMIT JOB COMPACT` command triggers the long time  `RocksDB compact` operation